### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by cph-sm522**

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -7,9 +7,8 @@
         <option value="$PROJECT_DIR$/pom.xml" />
       </list>
     </option>
-    <option name="workspaceImportForciblyTurnedOn" value="true" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" default="true" project-jdk-name="corretto-17" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_21" default="true" project-jdk-name="corretto-17" project-jdk-type="JavaSDK">
```
Comment (Scrabble30) [inline .idea/misc.xml]: Revert all changes made to misc.xml. The language level JDK and other changes cannot be allowed to be made in this file

File: src/main/java/app/controllers/OrderController.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package app.controllers;
+
+
```
Comment (Scrabble30) [inline src/main/java/app/controllers/OrderController.java]: Remove unnecessary space

File: src/main/java/app/controllers/OrderController.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import app.persistence.ConnectionPool;
+import io.javalin.Javalin;
+import io.javalin.http.Context;
+
+import java.sql.Connection;
```
Comment (Scrabble30) [inline src/main/java/app/controllers/OrderController.java]: Remove unused import

File: src/main/java/app/controllers/OrderController.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+public class OrderController {
+
+    public static void addRoutes(Javalin app, ConnectionPool connectionpool){
```
Comment (Scrabble30) [inline src/main/java/app/controllers/OrderController.java]: connectionpool parameter name needs to be changed to connectionPool to abide by camel case code standard

File: src/main/java/app/controllers/OrderController.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        app.post("addtobasket", ctx -> addToBasket(ctx, connectionpool));
+        app.post("myorders", ctx -> viewMyOrders());
+        app.post("basket", ctx -> viewMyBasket());
+        app.post("ordernow", ctx -> placeOrder());
+        app.post("cancelovervieworder", ctx -> cancelOrderInOverview());
```
Comment (Scrabble30) [inline src/main/java/app/controllers/OrderController.java]: Should routing be changed to cancelorderinoverview to be more clear?

**Review by Scrabble30 — CHANGES_REQUESTED** [review]

File: .idea/misc.xml (hunk 1 above)
Comment (cph-sm522) [reply .idea/misc.xml]: Har ikke foretaget nogen ændring her.. mit kører på 17

Comment (Scrabble30) [reply .idea/misc.xml]: misc.xml filen blev committet med ændringer i committen orderController

**Review by Scrabble30 — APPROVED** [review]

**Pull request merged**